server/src/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,24 @@
+import UserSerializer from "./UserSerializer.js"
+
+class ReviewSerializer {
+  static getSummary(review) {
+    const allowedAttributes = ["id", "description", "rating"]
+
+    let serializedReview = {}
+
+    for (const attribute of allowedAttributes) {
+      serializedReview[attribute] = review[attribute]
+    }
+    return serializedReview
+  }
+
+  static async updateData(reviews) {
```

Comment:
`updateData` isn't very clear what we're doing with this method, something like `getDetails` or `getUser` is more descriptive